namespace java com.github.ilms49898723.meichu.connection

service UserService {
    list<string> getUserList()
}
